src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,25 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
`post.user` might be null theoretically. It will cause an error when we try to read some properties of it. So we should handle this case. If there is no user do not render the `UserInfo` component at all (conditional rendering)